lib/http/http_service.dart

```diff
@@ -1,14 +1,13 @@
 import 'dart:convert';
 import 'package:flutter/foundation.dart';
-import 'package:hobby_hub_ui/models/network.dart';
 import 'package:hobby_hub_ui/models/pin.dart';
 import 'package:http/http.dart';
 
 // HELP with http related things:
 // https://flutter.dev/docs/cookbook/networking/fetch-data#2-make-a-network-request
 
 class HttpService {
-  final String ipUrl = "http://192.168.0.66:5000";
+  final String ipUrl = "http://192.168.1.179:5000";
```

Comment:
Okay for now, but will eventually use Andy's Get_IP class on backend to change to use WiFi IP address. I.E. not hardcoded.